.github/workflows/github-action.yaml

```diff
@@ -0,0 +1,73 @@
+# This is a sample github action to build and release your plugin using a github action
+# It will run when pushing a new tag - the tag name must be your plugin version number (e.g. "1.0.0.0")
+# Replace and adjust the areas that are commented below
+# Place it into the workflow folder of your repository at .github/workflows
+name: Build and Release
+
+# Every time a new tag with the typical assembly format is pushed this will run. e.g. tag name "1.0.0.0"
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+.[0-9]+'
+
+permissions:
+  contents: write
+
+env:
+  # Adjust this to your plugin title
+  PLUGIN_NAME: "Speckle Interferometry"
+  PLUGIN_SLN_NAME: "NINA.Plugin.Speckle"
+  PLUGIN_DLL_NAME: "SpeckleInterferometry"
+
+jobs:
+  build-and-release:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # In case you need more sub folders add these here
+      - name: Prepare folders
+        run: |
+          mkdir packages
+          mkdir "packages/${{ env.PLUGIN_NAME }}"
+
+      # This will build your solution. If the solution name differs from your plugin name, please adjust it here
+      - name: Build .NET Assemblies
+        run: |
+          dotnet restore NINA.Plugin.Speckle/${{ env.PLUGIN_SLN_NAME }}.sln
+          dotnet build "NINA.Plugin.Speckle/${{ env.PLUGIN_SLN_NAME }}.sln" -c Release -p:PostBuildEvent= -p:Version=${{ github.ref_name }}
+
+      # If you have mkdocs documentation you want to include, you can uncomment and build it like this
+      # - name: Build Documentation
+      #   run: |
+      #     python -m pip install --upgrade pip
+      #     pip install mkdocs
+      #     pip install mkdocs-material
+      #     mkdocs build -f ${{ env.PLUGIN_NAME }}\docs\mkdocs.yml
+
+
+      # Add all necessary files that the plugin needs to the packages folder - basically all items that are normally in your post build event on your local builds
+      - name: Prepare package
+        run: |
+          Copy-Item "NINA.Plugin.Speckle/NINA.Plugin.Speckle/bin/Release/net8.0-windows/${{ env.PLUGIN_DLL_NAME }}.dll" "packages/${{ env.PLUGIN_NAME }}/${{ env.PLUGIN_DLL_NAME }}.dll" -Force
+          Copy-Item "NINA.Plugin.Speckle/NINA.Plugin.Speckle/bin/Release/net8.0-windows/${{ env.PLUGIN_DLL_NAME }}.pdb" "packages/${{ env.PLUGIN_NAME }}/${{ env.PLUGIN_DLL_NAME }}.pdb" -Force
+      - name: Create Plugin archives and manifests
+        run: |
+          curl https://api.bitbucket.org/2.0/repositories/isbeorn/nina.plugin.manifests/src/main/tools/CreateNET7Manifest.ps1 >> CreateNET7Manifest.ps1
```

Comment:
Outdated URL. You can also use `curl -LO <url>` to directly save the fetched file as it is named in the URL instead of redirecting stdout to a file.